xonsh/procs/pipelines.py

```diff
@@ -175,8 +175,8 @@ def __init__(self, specs):
         self.proc = self.procs[-1]
 
     def __repr__(self):
-        s = self.__class__.__name__ + "("
-        s += ", ".join(a + "=" + repr(getattr(self, a)) for a in self.attrnames)
+        s = self.__class__.__name__ + "(\n  "
+        s += ",\n  ".join(a + "=" + repr(getattr(self, a)) for a in self.attrnames)
         s += ")"
```

Comment:
I think the closing parenthese should be on its own line, so this should be `\n)`